client/post-editor/editor-publish-date/README.md

```diff
@@ -0,0 +1,30 @@
+Editor Publish Date
+===================
+
+Editor Publish Date is a React component that allows the user to set a post's publish date inside the Editor.
+
+## Usage
+
+```jsx
+import EditorPublishDate from 'post-editor/editor-publish-date';
+
+export default function MyComponent() {
+	const props = {
+		post: post,
+		postDate: postDate,
+		setPostDate: setPostDate,
+	};
+
+	return (
+		<EditorPublishDate { ...props } />
+	);
+}
+```
+
+## Props
+
+The following props are used with the Editor Publish Date component:
+
+- `post`: (object) The post object
+- `postDate`: (string) The publish date for the post
```

Comment:
Why does the publish date get passed in separately instead of just using what is set on `post`?